test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1

```diff
@@ -59,18 +59,21 @@ Describe "Test-Connection" -tags "CI" {
             $result2 | Should -BeFalse
         }
 
-        It "Ping fake host" {
+        It 'returns false without errors for an unresolvable address when using -Quiet' {
+            $result = $true
+            { $result = Test-Connection -Quiet -ErrorAction Stop -Count 1 -TargetName "fakeHost" } | Should -Not -Throw
+            $result | Should -BeFalse
```

Comment:
Since we expect result I suggest simplify the test:
```suggestion
            Test-Connection -Quiet -ErrorAction Stop -Count 1 -TargetName "fakeHost" | Should -BeFalse
```